.github/workflows/build.yml

```
@@ -64,6 +64,22 @@ jobs:
             ${{ env.BASE_CACHE_KEY }}
       - name: Setup curl cache
         run: mkdir -p ${{ env.CURL_CACHE_DIR }}
+       # Install GoCrtitic, GoSec and Staticcheck Go tools 
+      - name: Install GoCritic 
```

Just aligning with how the tools are referred to by their creators.

```suggestion
       # Install go-crtitic, GoSec and staticcheck Go tools 
      - name: Install go-critic
```